### ISSUE TITLE
a11y review of page layout

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#468]
Provide feedback on what elements/roles should be used for constructing the layout of PF4 page designs.

As part of this, determine if redundant roles are necessary for HTML5 elements.

**Comment by jgiardino** [comment]
Information about implicit role mappings is included at the end of this article, along with links to related studies that were done:
https://a11yproject.com/posts/aria-landmark-roles/

These are old, so it would be good to test ourselves if redundant roles are still needed.

**Comment by jgiardino** [comment]
Just came across this recommendation to never set redundant roles: https://www.w3.org/TR/html5/dom.html#do-not-set

**Comment by virtual** [comment]
The Udacity course on a11y also mentions that landmarks would be the only exception to the rule (of not being redundant) for browser support: https://classroom.udacity.com/courses/ud891/lessons/8311490720/concepts/83024007410923

**Comment by jgiardino** [comment]
Closing this issue. Specific updates to the Page component landmark regions were captured in https://github.com/patternfly/patternfly-next/issues/1185
